osu.Game/Overlays/Settings/Sections/Maintenance/DirectorySelectScreen.cs

```diff
@@ -24,7 +24,7 @@ public abstract class DirectorySelectScreen : OsuScreen
 
         private OsuDirectorySelector directorySelector;
 
-        public override bool AllowTrackAdjustments => false;
+        public override bool? AllowTrackAdjustments => false;
```

Comment:
This `false` specification is now incorrect (should be inherit). Was added as a best-effort means.